website/content/en/preview/concepts/node-templates.md

```diff
@@ -60,6 +60,12 @@ Select subnets by name:
     Name: my-subnet
 ```
 
+Select subnets using comma separated tag values:
+```yaml
+  subnetSelector:
+    Name: "my-subnet-1,my-subnet-2"
```

Comment:
Hi, I would prefer to have something like :
```
subnetSelector:
  Name: [my-subnet-1,my-subnet-2
```
with the possibility to provide variables as list or a string for unique subnet.